ui/components/app/beta-header/index.js

```diff
@@ -28,8 +32,9 @@ const BetaHeader = () => {
       className="beta-header"
       alignItems={AlignItems.center}
     >
-      <Typography
-        variant={TypographyVariant.H7}
+      <Text
+        variant={TextVariant.bodySm}
+        as="h6"
         marginTop={0}
         marginBottom={0}
```

Comment:
The Text component doesn't have any default margins so you could remove these
```suggestion
```

Reply:
I have made the suggested changes.
Please review the commit, thanks!